images/ext-auth-example-authsvc/rootfs/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.17.6-alpine3.15 as builder
+RUN mkdir /authsvc
+WORKDIR /authsvc
+COPY . ./
+RUN CGO_ENABLED=0 GOOS=linux go build -o authsvc authsvc.go
+
+FROM alpine:3.10
```

Comment:
Please use latest alpine version or even better, distroless

Reply:
Sure @rikatz. Updated to distroless.